Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clean.py

```diff
@@ -45,11 +46,16 @@ def parser(cls, parser, loggers=None):
             '--remote', dest='remote', type=str, default=None,
             help='Specify remote to search for pull request from.',
         )
+        parser.add_argument(
+            '--pr-branches', dest='pr_branches', action='store_true', default=False,
```

Comment:
What exact command will users have to run to use this? Is it `git-webkit clean --pr-branches`? 
I would rather prefer just one parameter like `git-webkit --clean-pr-branches` or `git-webkit --delete-stale-pr-branches`. With `git-webkit clean --pr-branches` I would be worried thinking that clean command might clean something else as well, or what if I mistyped the second (--pr-branches) parameter, and clean command cleaned up something else. With just one parameter if I type that parameter incorrectly, I would simply get an error.

Reply:
FWIW, a new sub-command seems more natural to me, too. Maybe that's not the git way, though?

I'd suggest using similar terminology for similar things (clean/clean up/delete could get one thinking about what's the difference is).

Reply:
My rationale is less about `git`-isms than it is about sub-command bloat in general, since we already have 15 (with @facetothefate adding at 16th today), but given I was already on the fence, 2 contributors wanting this to be a sub-command makes that the obvious path in my mind now, so I'll make it a sub-command.